source/adapters/cuda/command_buffer.hpp

```diff
@@ -169,10 +170,10 @@ static inline const char *getUrResultString(ur_result_t Result) {
 #define UR_CALL(Call, Result)                                                  \
   {                                                                            \
     if (PrintTrace)                                                            \
-      fprintf(stderr, "UR ---> %s\n", #Call);                                  \
+      logger::always("UR ---> {}", #Call);                                     \
     Result = (Call);                                                           \
     if (PrintTrace)                                                            \
-      fprintf(stderr, "UR <--- %s(%s)\n", #Call, getUrResultString(Result));   \
+      logger::always("UR <--- {}({})\n", #Call, getUrResultString(Result));    \
```

Comment:
Should `\n` have been removed here? Or should it also have been removed on line 176?

Reply:
done